### ISSUE TITLE
Magit 2.1 does not play well with Evil

### ISSUE THREAD

**Issue by smondet** [magit/magit#1964]
I don't know how it happened but the new Magit 2.1 really doesn't play well with Evil-mode
https://bitbucket.org/lyro/evil/issue/535/magit-21-takes-over-normal-mode

(and the PR https://bitbucket.org/lyro/evil/pull-request/16/adapt-for-magit-v210/diff 
referenced in https://github.com/magit/magit/issues/1921
has no impact since I explicitly `nil` that variable, I am talking about Evil's normal mode, not emacs-mode).

**Comment by kyleam** [comment]
Do the changes mentioned in #1950 help?

```
(add-to-list 'evil-emacs-state-modes 'magit-popup-mode)
(add-to-list 'evil-emacs-state-modes 'magit-popup-sequence-mode)
```

**Comment by kyleam** [comment]
(Oops, nevermind.  I see that the PR you referenced was in response to that issue.)

**Comment by smondet** [comment]
These lines add magit modes to the list of modes that Evil starts in Emacs mode. I don't want that behavior. I want always normal-mode.

The problem is that even in Evil's normal-mode, Magit bindings take over Evil's ones.

E.g. I ended up loosing work because I typed `kyy` to go up an copy a line in the magit buffer, but instead it deleted a not-yet-staged file.
This used to work fine with Magit 1.x (and it seems difficult to install older version of Emacs packages …).


Related, is there a way to disable *all* Magit key bindings?
